apps/spruce/src/components/FilterChips/FilterChip.tsx

```diff
@@ -9,20 +9,25 @@ const maxChipLength = 25;
 
 interface FilterChipType {
   key: string;
-  value: string;
   title: string;
+  value?: string;
```

Comment:
I think it should be the opposite; title should be optional, value should be required